packages/app/controllers/auth.js

```diff
@@ -117,12 +146,20 @@ async function signinController(req, res, next) {
           message: Messages.INCORRECT_EMAIL_PASS,
           statusCode: 404,
         });
+
       if (!user.is_verified)
         return res.status(403).json({
           error: STATUS_CODES[403],
           message: Messages.EMAIL_NOT_VERIFIED,
           statusCode: 403,
         });
+      if (user.is_deleted) {
+        return res.status(400).json({
+          error: STATUS_CODES[400],
+          message: "Account does not exist.",
```

Comment:
How does this message sounds?

```suggestion
          message: "Email not available.",
```